scripts/linux/publishReleasePackages.sh

```diff
@@ -127,12 +133,39 @@ process_args() {
             "-b" | "--branch-name") save_next_arg=8 ;;
             "-pro" | "--pmc-repository") save_next_arg=9 ;;
             "-pre" | "--pmc-release") save_next_arg=10 ;;
+            "--setup-pmc-only") save_next_arg=11 ;;
             *) usage ;;
             esac
         fi
     done
 }
 
+#######################################
+# NAME: 
+#    publish_to_microsoft_repo
+#
+# DESCRIPTION:
+#    The function setup the secrets and config file for RepoClient app.
+#######################################
+setup_for_microsoft_repo()
+{
+#Cleanup
+sudo rm -rf $WDIR/private-key.pem || true
+sudo rm -f $SETTING_FILE || true
+
+#Download Secrets - Requires az login and proper subscription to be selected
+az keyvault secret download --id 'https://edgereleasekv.vault.azure.net/secrets/iotedge-pmc-client-auth-prod' \
+    --subscription IOT_EDGE_DEV1 \
+    -o tsv \
+    --query 'value' \
+    --encoding base64 \
+    -f ./iotedge-pmc-client-auth-prod.pfx
```

Comment:
In other pipelines, we've always used the Key Vault task (which uses a service connection, so we don't have to hard code information about our internal subscriptions, Key Vault URLs, etc.), and then passed the secrets to the script as env vars.